.github/workflows/build-images.yaml

```diff
@@ -411,30 +411,30 @@ jobs:
           DF_PATH: "./benchmarks/${{ github.job }}/${{ matrix.tag }}"
           DBX_PLATFORM: ${{ matrix.platform }}
 
-  fb-oss-performance:
-    runs-on: ubuntu-latest
-    needs: [mysql, siege, hhvm]
-    env:
-      DH_REPO: "cloudsuite/${{ github.job }}"
-    strategy:
-      matrix:
-        tag: ["4.0"]
-        platform: ["linux/amd64"]
-    steps:
-      - name: checkout
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-      - if: ${{ needs.mysql.outputs.is_parent_modified == 'true' ||
-          needs.siege.outputs.is_parent_modified == 'true' ||
-          needs.hhvm.outputs.is_parent_modified == 'true' }}
-        run: echo "IS_PARENT_MODIFIED=true" >> $GITHUB_ENV
-      - name: build and push
-        run: "./.github/scripts/build-images.sh"
-        env:
-          IMG_TAG: "${{ matrix.tag }}"
-          DF_PATH: "./benchmarks/${{ github.job }}/${{ matrix.tag }}"
-          DBX_PLATFORM: ${{ matrix.platform }}
+  # fb-oss-performance:
+  #   runs-on: ubuntu-latest
+  #   needs: [mysql, siege, hhvm]
+  #   env:
+  #     DH_REPO: "cloudsuite/${{ github.job }}"
+  #   strategy:
+  #     matrix:
+  #       tag: ["4.0"]
+  #       platform: ["linux/amd64"]
+  #   steps:
+  #     - name: checkout
+  #       uses: actions/checkout@v2
+  #       with:
+  #         fetch-depth: 0
+  #     - if: ${{ needs.mysql.outputs.is_parent_modified == 'true' ||
+  #         needs.siege.outputs.is_parent_modified == 'true' ||
+  #         needs.hhvm.outputs.is_parent_modified == 'true' }}
+  #       run: echo "IS_PARENT_MODIFIED=true" >> $GITHUB_ENV
+  #     - name: build and push
+  #       run: "./.github/scripts/build-images.sh"
+  #       env:
+  #         IMG_TAG: "${{ matrix.tag }}"
+  #         DF_PATH: "./benchmarks/${{ github.job }}/${{ matrix.tag }}"
+  #         DBX_PLATFORM: ${{ matrix.platform }}
```

Comment:
If we deprecate the workload, do we need to keep these lines as comments in the script? It makes it ugly and does not serve any purpose if we won't build them again in the future.